lib/models/device.coffee

```diff
@@ -201,6 +201,32 @@ exports.isOnline = (uuid, callback) ->
 	exports.get(uuid).get('is_online').nodeify(callback)
 
 ###*
+# @summary Get the local IP addresses of a device
+# @name getLocalIPAddresses
+# @public
+# @function
+# @memberof resin.models.device
+#
+# @param {String} uuid - device uuid
+# @returns {Promise<Array<String>>} local ip addresses
+#
+# @throws Will throw if the device is offline.
+#
+# @example
+# resin.models.device.getLocalIPAddresses('7cf02a62a3a84440b1bb5579a3d57469148943278630b17e7fc6c4f7b465c9').then (localIPAddresses) ->
+# 	for localIP in localIPAddresses
+# 		console.log(localIP)
+###
+exports.getLocalIPAddresses = (uuid, callback) ->
+	exports.get(uuid).then (device) ->
+		if not device.is_online
```

Comment:
Yeah, it's just that it can be confusing to users (and the ip address reported will be correct in the case of having connectivity but no vpn)